lib/local-cluster/process/Cardano/Wallet/Launch/Cluster/Process.hs

```diff
@@ -0,0 +1,258 @@
+    port <- liftIO getRandomPort
+    faucetFundsPath <- ContT withTempFile
+    liftIO $ saveFunds (FileOf $ absFile faucetFundsPath) faucetFundsValue
+    (logsPathName, command) <-
+        localClusterCommand name walletOption envs port faucetFundsPath
+    ToTextTracer processLogs <- case logsPathName of
+        Nothing -> pure $ ToTextTracer nullTracer
+        Just path ->
+            ContT
+                $ newToTextTracer
+                    (path <> "-process" <.> "log")
+                    Nothing
+    _ <-
+        ContT
+            $ withBackendProcess
+                processLogs
+                command
+                NoTimeout
+                DoNotSendSigINT
+    queries <- withSNetworkId (NTestnet 42)
+        $ \network -> withServiceClient network port nullTracer
+    pure (queries, ToTextTracer processLogs)
+
+-- | Generate a command to start a local cluster with the given the local cluster
+-- name, monitoring port, and faucet funds path.
+localClusterCommand
+    :: FilePath
+    -- ^ filename to append to the logs dir
+    -> WalletPresence
+    -- ^ whether to start the wallet
+    -> EnvVars
+    -- ^ environment variables that have to be used
+    -> PortNumber
+    -- ^ monitoring port
+    -> FilePath
+    -- ^ faucet funds path
+    -> ContT r IO (Maybe FilePath, Command)
+localClusterCommand name walletOptions envs port faucetFundsPath = do
+    configsPath <- liftIO $ getClusterConfigsPathFromEnv envs
+    mLogsPath <- liftIO $ getClusterLogsFilePathFromEnv envs
+    mMinSeverity <- liftIO $ getClusterLogsMinSeverity envs
+    (clusterStdout, logsPathName) <- case mLogsPath of
+        Nothing -> pure (NoStream, Nothing)
+        Just logsPath -> do
+            let logsPathName = logsPath </> name
+            fmap (\h -> (UseHandle h, Just logsPathName))
+                $ ContT
+                $ withFile (logsPath </> name <> "-stdout" <.> "log") WriteMode
+
+    pure
+        $ (logsPathName,)
+        $ Command
+            { cmdName = "local-cluster"
+            , cmdArgs =
+                [ "--faucet-funds"
+                , faucetFundsPath
+                , "--monitoring-port"
+                , show port
+                , "--cluster-configs"
+                , configsPath
+                ]
+                    <> case mLogsPath of
+                        Nothing -> []
+                        Just logsPath ->
+                            [ "--cluster-logs"
+                            , logsPath </> name <.> "log"
+                            ]
+                    <> case mMinSeverity of
+                        Nothing -> []
+                        Just minSeverity ->
+                            [ "--min-severity"
+                            , show minSeverity
+                            ]
+                    <> case walletOptions of
+                        NoWallet -> []
+                        WalletPresence mPortNumber ->
+                            ["--wallet-present"]
```

Comment:
See other comment.

Reply:
The only good reason why we want it now is that wallet-e2e are using it